jupyterhub_config.py

```diff
@@ -1,13 +1,110 @@
 from jupyterhub.spawner import SimpleLocalProcessSpawner
 
+from jhub_apps import theme_template_paths
 from jhub_apps.configuration import install_jhub_apps
 
 c = get_config()  # noqa
 
 from jupyterhub.auth import DummyAuthenticator  # noqa: E402
 
+c.Authenticator.admin_users = {"admin"}
 c.JupyterHub.authenticator_class = DummyAuthenticator
 c.JupyterHub.log_level = 10
 
-c.JupyterHub.bind_url = "http://127.0.0.1:8000"
+hub_url = "http://127.0.0.1:8000"
+
+c.JupyterHub.bind_url = hub_url
 c = install_jhub_apps(c, spawner_to_subclass=SimpleLocalProcessSpawner)
+
+c.JupyterHub.default_url = "/hub/home"
```

Comment:
This ensures that a user is redirected to the homepage when they click the top right logo, are are directed to the default_url by some JupyterHub mechanism